### DISCUSSION TITLE
Accessing data via ExtJS

### DISCUSSION THREAD

**Discussion by alexanderkroneis** [pimcore/pimcore#10669] [Q&A / Support] [answered]
Hi,

I'm currently working on a project and our customer needs some modification within pimcore backend. I registered the `postOpenObject` event via JavaScript and need to access the data within the handler. How?

**Comment by dvesh3** [comment]
@alexgall please see this example in the documentation https://pimcore.com/docs/pimcore/current/Development_Documentation/Best_Practice/Adding_Button_To_Object_Editor.html

**Reply by alexanderkroneis** [reply]
Hi, I added my button to the editor but now I need to access the data within my handler.

```
postOpenObject: function (object, type) {
    if (object.data.general.o_key === 'Member' && type === 'folder') {
        object.toolbar.add({
            text: t('Start Survey'),
            iconCls: '',
            handler: function (object) {
            }.bind(this, object)
        })

        pimcore.layout.refresh()
    }
},
```

**Comment by alexanderkroneis** [comment, marked as answer]
It's object.search.grid.getSelectionModel().getSelection() by the way.
